### PR TITLE
Adds a validator for checking if a string has text and tests for this validator

### PR DESCRIPTION
**Pull request by rubenisgurkitis**

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    validators.HAS_TEXT("\n     ").should.eql(false)
+  })
+  it('should return false for a string made of tabs and line breaks', function() {
+    validators.HAS_TEXT("\t\n\n\n\n\n\n\n").should.eql(false)
+  })
+  it('should return false for a string made of invisible characters', function() {
+    validators.HAS_TEXT("\n\t\n         ").should.eql(false)
+  })
+  it('should return true for a string made only of letters', function() {
+    validators.HAS_TEXT("QueensOfTheStoneAge").should.eql(true)
```
Comment (chrisdoc) [inline test/test.js:18]: @rubenisgurkitis great taste in music

Comment (rubenisgurkitis) [inline test/test.js:18]: @chrisdoc Just what I was listening in that moment, but yes, they're awesome.

**Comment by rubenisgurkitis** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;b1MUfWRP&#x2F;33-bug-patient-can-send-empty-messages-using-white-space-or-new-line-characters-to-doctor-0-25"> [BUG] Patient can send empty messages (using white space or new line characters) to doctor [0.25]</a>

**Comment by rjoonas** [comment]
@rubenisgurkitis Just a friendly heads up, this is a public repo so linking Trello cards here seems to make their title visible to the general public even if the Trello board itself is private.

**Pull request merged**